assets/stylelint.config.mjs

```diff
@@ -38,5 +39,9 @@ export default {
     // Conflicts with Prettier, which breaks long lines involving operators by
     // putting operators at the ends of lines.
     "scss/operator-no-newline-after": null,
+    "plugin/no-unsupported-browser-features": [
+      true,
+      { ignore: ["intrinsic-width"] },
```

Comment:
If we ignore something we should probably add a note explaining why. I assume it's because the only places we use this feature are on screen types that actually do support intrinsic-width?

Reply:
Before adding the correct Bus E-Ink version, this was the only error we needed to deal with and the browser it is an issue for doesn't reach the particular CSS in question. After the update, there's a 919 errors so adding global ignores isn't a good idea. I'm curious how best to handle this. At a webpack level, that seems as easy as separating out the apps into their own entries and specifically calling out the browser each entry will use. I'm not sure about the linters though...

Reply:
Hm... is this warning us about using features that Babel will ultimately compile down to something that does actually work? Or are those warnings truly about features that can't be supported automatically?

Reply:
Follow-up question for investigation, does the Webpack build itself throw an error or warning when we use a feature Babel can't transpile to the target browsers, or does it silently allow such features?

Reply:
Hard to tell at the moment if it is failing silently or if it just so happens that our code can transpile for all configured browsers. I tried dropping the target down to `Chrome 4` and still see no errors.

Reply:
Yeah that sounds good. Will remove those and create a task.

Reply:
I'm curious if we should do the same with ESLint. The lint errors showing up right now are either false positives or handled by the polyfills we are already importing that ESLint has no knowledge of.

Reply:
Interesting. I would have expected the opposite based on [this thread](https://github.com/amilajack/eslint-plugin-compat/issues/602), which warns that the ESLint plugin is _too_ lenient in assuming which polyfills are available (suggesting there is some interaction with Babel).

Reply:
I may have just configured the plugin incorrectly (documentation is lackluster unfortunately). I made an adjustment and the errors are gone. I found in the [source code](https://github.com/amilajack/eslint-plugin-compat/blob/a0c853e260656cb60139f90e3ffc89d025f80969/src/rules/compat.ts#L91) for the plugin that it assumes polyfills are done in babel based on the presence of a babel config file. We didn't have one so I made it. Hopefully this doesn't mean I just silenced the errors by accident.